base/applications/rapps/unattended.cpp

```diff
@@ -80,12 +87,19 @@ HandleInstallCommand(CAppDB *db, LPWSTR szCommand, int argcLeft, LPWSTR *argvLef
         }
     }
 
-    return DownloadListOfApplications(Applications, TRUE);
+    return DownloadListOfApplications(Applications, flags);
 }
 
 static BOOL
 HandleSetupCommand(CAppDB *db, LPWSTR szCommand, int argcLeft, LPWSTR *argvLeft)
 {
+    UINT flags = DAF_MODAL;
+    if (argcLeft >= 1 && !StrCmpIW(L"/S", argvLeft[0]))
+    {
+        flags |= DAF_SILENT;
+        argvLeft = &argvLeft[1], --argcLeft;
+    }
+
     if (argcLeft != 1)
```

Comment:
Looking at the other `HandleInstallCommand` function, should this stay `if (argcLeft != 1)` or be changed to `if (argcLeft < 1)` ?